tools/deploy-to-svn.sh

```diff
@@ -66,12 +75,12 @@ done
 echo "Done!"
 
 # Tag the release.
```

Comment:
Perhaps the only thing I'd add here is a bail out in case `$TARGET` is `HEAD`. You know, in case someone decides to uncomment the following lines in the future.